stable/lamp/templates/pvc.yaml

```diff
@@ -0,0 +1,24 @@
+{{- if and .Values.persistence.enabled (not .Values.persistence.hostPath) -}}
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+  {{- if .Values.persistence.storageClass }}
+  annotations:
+    volume.beta.kubernetes.io/storage-class: {{ .Values.persistence.storageClass | quote }}
```

Comment:
No longer use annotation for storage class. We've developed a pattern for persistence (see #1869). Please follow this approach.